evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -139,6 +139,19 @@ def test_init_str_components():
     assert comp_graph.compute_order == expected_order
 
 
+def test_init_instantiated():
+    graph = {
+        "Imputer": [
+            Imputer(numeric_impute_strategy="constant", numeric_fill_value=0),
+            "X",
+            "y",
+        ]
+    }
+    cg = ComponentGraph(graph)
+    cg.instantiate({"Imputer": {}})
+    assert graph["Imputer"][0] == cg.get_component("Imputer")
```

Comment:
Do we have a test that the component instance's set parameters are still kept in the component graph / pipeline?

Reply:
Updated test to make sure that existing parameters are not changed during instantiation.